Garbled Circuit.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Garbled Circuit
+parent: Protocols
+grand_parent: Contents
+has_children: true
+nav_order: 1
+---
+
+# Secure Multi-Party Computation from Garbling
+
+
+Author: Xiang Xie @PlatON
+
+This post was initially published in [PlatON](https://www.platon.network/news/4?newsType=2).
+
+As described in the previous article, secure multi-party computation protocols are distributed protocols that allow participated players to jointly compute some pre-agreed function without revealing the privacy of each input information. PlatON provides an end-to-end MPC framework to enable privacy-preserving computation. In this article, we will get into more details on the techniques to show how it works.

```

Comment:
There is no previous article. Remove

Reply:
"PlatOn provides a n end-to-end MPC framework"  - should be deleted

Reply:
Agree.